python/lsst/rubintv/production/starTracker.py

```diff
@@ -130,27 +130,75 @@ def dayObsToDateTime(dayObs):
     return datetime.datetime.strptime(dayObsIntToString(dayObs), '%Y-%m-%d')
 
 
+def isStreamingModeFile(filename):
+    """Check if a filename is a streaming mode file.
+
+    Parameters
+    ----------
+    filename : `str`
+        The filename.
+
+    Returns
+    -------
+    isStreaming : `bool`
+        Whether the file is a streaming mode file.
+    """
+    return len(os.path.basename(filename).split('_')) == 5
```

Comment:
Thanks, I've pushed a comment with example filenames. And just so you know, all this will be moving to be butler driven "soon" so all the filename stuff will become moot at that point anyway.